app/uk/gov/hmrc/eeittadminfrontend/binders/ValueClassBinders.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eeittadminfrontend.binders
+
+import play.api.libs.json.{JsError, JsString, JsSuccess, Reads}
+import play.api.mvc.PathBindable
+import uk.gov.hmrc.eeittadminfrontend.models.FormTypeId
+
+/**
```

Comment:
As I now we don't need provide this intellij comments. Everything is in git history :) How about removing it? Less code to read.